pom.xml

```diff
@@ -163,12 +164,17 @@
             <dependency>
                 <groupId>io.opentracing</groupId>
                 <artifactId>opentracing-api</artifactId>
-                <version>0.31.0</version>
+                <version>${opentracing.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.opentracing</groupId>
+                <artifactId>opentracing-noop</artifactId>
+                <version>${opentracing.version}</version>
             </dependency>
             <dependency>
                 <groupId>com.uber.jaeger</groupId>
                 <artifactId>jaeger-core</artifactId>
-                <version>0.23.0</version>
+                <version>1.1.1-1136-e7ae7fd51c4</version>
```

Comment:
We should migrate this to `io.jaegertracing:jaeger-core:0.35.5`, which is the latest OSS Jaeger version

Reply:
tchannel shouldn't even have this dependency, only opentracing